src/index.js

```diff
@@ -0,0 +1,98 @@
+import './style.css';
+import add from './modules/add.js';
+import remove from './modules/delete.js';
+import { editIndex, checkboxFunction } from './modules/utilities.js';
+
+const listContainer = document.querySelector('.listContainer');
+const inputText = document.querySelector('#input');
+const addBtn = document.querySelector('.submit');
+const clearAllBtn = document.querySelector('.completed_all');
+
+// Adding new task;
+
+let tasks = [];
+
+if (localStorage.getItem('to-do') === null) {
+  tasks = [];
+} else {
+  tasks = JSON.parse(localStorage.getItem('to-do'));
+}
+const setLocalStorage = () => {
+  localStorage.setItem('to-do', JSON.stringify(tasks));
+};
+
+const displayList = () => {
+  listContainer.innerHTML = '';
+  let html = '';
+  tasks.forEach((task, i) => {
+    html += `<div class="items" id = "${i}">
+        <li class="li">
+          <input type="checkbox" class="checkbox" ${
+  task.completed ? 'checked' : ''
+}>
+          ${task.description}
+        </li>
+        <div class="icons">
+        <i id="edit${i}" class="fas fa-edit hidden"></i>
+        <i id = "trash${i}" class="fas fa-trash-alt hidden"></i>
+        <i id = "ellipsis" class="fas fa-ellipsis-v"></i>
+        </div>
+      </div>`;
+    listContainer.innerHTML = html;
+  });
+};
+displayList();
+editIndex(tasks);
+
+const items = document.querySelectorAll('.items');
```

Comment:
- I think on line 47 you are adding an event listener for items class why you did not add event listen when you generating items list.
- it that time you will reduce search item 
- you will use only one loop for adding event listener and creating Dom elements.